pages/members/Member.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import styles from "./members.module.css"
+import Image from "next/image"
+import bannerImgDefault from "../../public/blank_banner.png"
+
+export default function Member(props) {
+ 
+  const profile_image = props.profile_image_url.replace("_normal", "")
+  const description = props.description.replace(/\n/g, ' ').trim()
+
+  const [profileLoad, setProfileLoad] = React.useState(false)
```

Comment:
Use `import React, { useState } from 'react';` to avoid `React.useState`

Reply:
changed to `import { useState } from "react"`